application/ui/src/app/pages/matching-group/matching-group.page.ts

```diff
@@ -115,10 +197,12 @@ export class MatchingGroupPage implements OnInit {
 
   clearGroup1() {
     this.selectComponentGroup1.clear();
+    this.selectComponentGroup1.close();
   }
 
   clearGroup2() {
     this.selectComponentGroup2.clear();
+    this.selectComponentGroup2.close();
```

Comment:
are you sure here when we close you can no more access the component and you have console error, please check and remove if we have those errors in the console

Reply:
yes don't worry about it. it just permit to close the modal ui. i have checked a console and nothing wrong